src/actions/clusterActions.js

```diff
@@ -174,11 +174,16 @@ export function clustersLoad() {
     );
 
     // Get the details for v5 clusters.
-    const v5ClustersDetails = await Promise.all(
+    let v5ClustersDetails = await Promise.all(
```

Comment:
fyi @axbarsan , I pushed this commit after you approved.

Can you take a look again sorry. I just deployed the branch and found an underlying related error.

Reply:
Will merge, assuming its ok.